Doxyfile.in

```diff
@@ -631,7 +631,7 @@ EXCLUDE_SYMBOLS        =
 # directories that contain example code fragments that are included (see
 # the \include command).
 
-EXAMPLE_PATH           = tutorial
+EXAMPLE_PATH           = @CMAKE_SOURCE_DIR@
```

Comment:
No because ``src/Mainpage.h`` uses ``example tutorial/lesson_01_basics.cpp``. Using ``tutorial`` caused lots of warnings to be emitted about this by Doxygen